### PR TITLE
Add stackLabels property into HighchartsAxisOptions interface

### PR DESCRIPTION
**Pull request by cyrilgandon**
Add stackLabels property into HighchartsAxisOptions interface
- Documentation: http://api.highcharts.com/highcharts/yAxis.stackLabels

Note that this is a property that only exists on yAxis. Maybe further work is too split yAxis and xAxis propertis in two different interface by sharing maybe a base interface.

### CODE REVIEW

**Comment by vvakame** [comment]
We can't get response from author. please ping me after 3 days. I'll merge this PR.

File: highcharts/highcharts.d.ts [hunk 1]
```diff
@@ -717,6 +717,82 @@ interface HighchartsAxisOptions {
...
+         * @default 0
+         * @since 2.1.5
+         */
+        rotation?: number;
+        /**
+         * CSS styles for the label.
+         * @default { "color": "#000000", "fontSize": "11px", "fontWeight": "bold", "textShadow": "0 0 6px contrast, 00 3px contrast" }
+         * @since 2.1.5
+         */
+        style?: Object;
```
Comment (baltie) [inline highcharts/highcharts.d.ts]: This should be the HighchartsCSSObject, not Object.

**Review by baltie — CHANGES_REQUESTED** [review]
Please change the style type. Apart from that, this looks fine.

**Comment by cyrilgandon** [comment]
Yes, of course, I used an example on another style property, and I got the `Object`. But most of the time it is `HighchartsCSSObject` which is far better. Thank you I've done the update.
I will try to find the time to change all the `Object` to `HighchartsCSSObject` where it make sense on another PR.

**Comment by baltie** [comment]
@cyrilgandon Thank you! :)
@dt-bot LGTM :+1:

**Review by baltie — APPROVED** [review]

**Comment by vvakame** [comment]
@cyrilgandon please resolve conflict.

**Comment by cyrilgandon** [comment]
@vvakame : conflicts have been resolved.

**Comment by cyrilgandon** [comment]
@baltie : I think I have made a little error, I plan to use TypeScript2.0 and the @types typings, and I just realised that the master branch is for legacy typings, I should have made a PR on the types-2.0 branch instead.
So I'm going to send another PR for that!

**Comment by baltie** [comment]
@cyrilgandon Unless I'm very mistaken, I think changes are merged periodically from master to the types-2.0 branch, so a separate PR for types-2.0 should be unnecessary.

**Comment by cyrilgandon** [comment]
@baltie : that's what is was supposed to be at the beginning of types-2.0, but I read that's that will not be the case.

https://github.com/DefinitelyTyped/DefinitelyTyped/issues/10711

> The merge from master to types-2.0 is manual, so "bad" merges in that direction shouldn't be happening. If you see this happen, let us know.

https://github.com/DefinitelyTyped/DefinitelyTyped/issues/11210

>We can merge master into types-2.0, but We can't reverse it before TypeScript 2.0 stable release.
If we merge PR into master and types-2.0 branch both. Merge operation require veeeeery high calories.
I want to avoid it.

If you look at the history, the last merged happens a long time ago, so I don't want to depend on future decision, moreover, for me at least, it is not clear what  will happen to master and types-2.0.
In doubt, I have made the PR on the types-2.0

**Comment by baltie** [comment]
@cyrilgandon Hm, I think we read that differently. As I understood what I've been reading, there is a merge performed manually by Microsoft engineers periodically. Also, the last merge from master to types-2.0 for highcharts (and a lot of other libraries at that point) were done by @mhegazy eight days ago. Which I think was the last one before the merge that happened yesterday my local time.

@vvakame Do you have any insight, should we make separate types-2.0 PRs? Also, I think a lot of this confusion would be avoided if the contributing.md file were updated with information about this. :)

**Comment by vvakame** [comment]
sorry, I don't have good insight.
I think `types-2.0` have a lot of different from `master` branch.
Microsoft team is resolving it by a human power.

**Pull request merged**